+++
title = "python"
date = "2008-04-12"
+++
* Author: vmiklos
* Vulnerable: 2.5.2-1
* Unaffected: 2.5.2-2kalgan1


David Remahl has discovered a security issue in the imageop module for Python, which can be exploited by malicious people to cause a DoS (Denial of Service) and potentially compromise a vulnerable system.
 The security issue is caused due to an incomplete fix (see FSA295) and can be exploited to cause a heap-based buffer overflow when specially crafted parameters are passed to the function.
 Successful exploitation may allow execution of arbitrary code.
* Bug Tracker URL: http://bugs.frugalware.org/task/2954
## CVEs:
* http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-1679
